src/models/query.js

```diff
@@ -0,0 +1,75 @@
+import find from'lodash.find';
+import map from'lodash.map';
+
+
+class Query {
+
+    constructor(name = '') {
+      this._aggregates = this._aggregates || [];
+      this._query = this._query || [];
+      this._query.fields = this._query.fields || [];
+      this.name = name;
+    }
+    // aggregates
+    addAggregate(aggregate) {
+      // @TODO validate QueryAggregate instance
+      this._aggregates = this._aggregates.concat(aggregate);
+      return this;
+    }
+    getAggregate(name) {
```

Comment:
this method doesn't exist anymore -> its now `getAggregates` easy simple. Getting by name wasn't used